var/spack/repos/builtin/packages/datatransferkit/package.py

```diff
@@ -56,6 +59,10 @@ def cmake_args(self):
             "-DDataTransferKit_ENABLE_EXAMPLES=OFF",
             "-DCMAKE_CXX_EXTENSIONS=OFF",
             "-DCMAKE_CXX_STANDARD=14",
+            "-DCMAKE_C_COMPILER=" + spec["mpi"].mpicc,
+            "-DCMAKE_CXX_COMPILER=" + spec["mpi"].mpicxx,
+            "-DCMAKE_Fortran_COMPILER=" + spec["mpi"].mpifc,
+            "-DMPI_BASE_DIR=" + str(pathlib.PurePosixPath(spec["mpi"].prefix)),
```

Comment:
Why do we need `pathlib.PurePosixPath`? Can't we just `str(spec["mpi"].prefix)`?

Reply:
I just copied this over from `trilinos/package.py`

perhaps @masterleinad or @aprokop  can comment on what DTK requires here..

Reply:
I see pathlib.PurePosixPath() was added in #34622

Removing it for now - so that this PR can be merged.